cli/commands/run/flags.go

```diff
@@ -165,7 +164,6 @@ func NewFlags(l log.Logger, opts *options.TerragruntOptions, prefix flags.Prefix
 			Negative:    true,
 			Destination: &opts.AutoInit,
 		},
-			flags.WithDeprecatedNames(terragruntPrefix.FlagNames("no-auto-init"), terragruntPrefixControl),
```

Comment:
should we include `no-auto-init`, `no-auto-retry` and `no-auto-approve` in the #4671 table?

Reply:
Will add after we merge that one